docs/sources/tempo/traceql/_index.md

```diff
@@ -204,52 +198,100 @@ Find traces where the average duration of the spans in a trace is greater than `
 avg(duration) > 20ms
 ```
 
-For example, find traces that have more than 3 spans with an attribute `http.status` with a value of `200`:
+For example, find traces that have more than 3 spans with an attribute `http.status_code` with a value of `200`:
 
 ```
-{ span.http.status = 200 } | count() > 3
+{ span.http.status_code = 200 } | count() > 3
 ```
 
 ## Arithmetic
 
 TraceQL supports arbitrary arithmetic in your queries. This can be useful to make queries more human readable:
 ```
-{ span.bytesProcessed > 10 * 1024 * 1024 }
+{ span.http.request_content_length > 10 * 1024 * 1024 }
 ```
 to compare the ratios of two span attributes:
 ```
-{ span.bytesProcessed < span.jobsProcessed * 10 }
+{ span.bytes_processed < span.jobs_processed * 10 }
 ```
 or anything else that comes to mind.
 
 ## Examples
 
-Find any trace with a span attribute or resource attribute `namespace` set to `prod`:
+### Simple query to find traces of a specific operation
+
+```
+{resource.service.name = "frontend" && name = "POST /api/orders"}
+```
+
+When using the same Grafana stack for multiple environment (e.g. `production`and `staging`) or having services that share the same name but are differentiated though their namespace, the query looks like:
+
+```
+{
+  resource.service.namespace = "ecommerce" &&
+  resource.service.name = "frontend" &&  
+  resource.deployment.environment = "production" && 
+  name = "POST /api/orders"
+}
+```
+
+### Query to find traces having a particular outcome
+
+All traces on the operation `POST /api/orders` that have an erroneous root span:
 
 ```
-{ .namespace = "prod" }
+{
+  resource.service.name="frontend" && 
+  name = "POST /api/orders" && 
+  status = error
+}
 ```
 
-Find any trace with a resource attribute `namespace` set to `prod`:
+All traces on the operation `POST /api/orders` that return with an HTTP 5xx error:
 
 ```
-{ resource.namespace = "prod" }
+{
+  resource.service.name="frontend" && 
+  name = "POST /api/orders" && 
+  span.http.status_code >= 500
+}
 ```
 
-Find any trace with a `namespace` attribute set to `prod` and `http.status` attribute set to `200`:
+### Query to find traces that have a particuliar behavior
+
+Query filtering on multiple spans of the traces.
+All the traces of the `GET /api/products/{id}` operation that access the database. It's a convenient request to identify caching problems:
+
+```
+{span.service.name="frontend" && name = "GET /api/products/{id}"} && {.db.system="postgresql"}
+```
+
+### Query to find traces going through a mix of "production" and "staging" instances
+
+Very convenient to identify misconfigurations and leaks across environments. 
+Find traces that go through "production" and "staging" instances.
+
+```
+{ resource.deployment.environment = "production" } && { resource.deployment.environment = "production" }
```

Comment:
The heading and text indicate that this is for searching through production and staging, but both of values in quotes are production. Should one of these be staging?